src/mtool/ui/src/components/VolumeManagement/VolumeList/index.js

```diff
@@ -289,11 +289,11 @@ class VolumeList extends Component {
         cellStyle: cellText
       },
       {
-        title: 'NQN',
+        title: 'UUID (NQN)',
```

Comment:
- Can you explain why this commit changed the table header from NQN to UUID?
- Why the title includes both UUID and NQN?

Reply:
That answers. Thanks. 👍